modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java

```diff
@@ -300,6 +301,27 @@ public void deleteObjectRelationshipMappingTableValues(
 					pkObjectFieldDBColumnNames.get(
 						"pkObjectFieldDBColumnName2"),
 					" = ", primaryKey2));
+
+			return;
+		}
+
+		ObjectField objectField2 = _objectFieldLocalService.getObjectField(
+			objectRelationship.getObjectFieldId2());
+
+		if (objectDefinition2.isSystem()) {
+			_objectEntryLocalService.insertIntoOrUpdateExtensionTable(
+				objectRelationship.getObjectDefinitionId2(), primaryKey2,
+				HashMapBuilder.<String, Serializable>put(
+					objectField2.getName(), 0L
+				).build());
+		}
+		else {
+			_objectEntryLocalService.updateObjectEntry(
```

Comment:
Question: Should we protect this code under an if clause for the TYPE_ONE_TO_MANY relationship type? I see that there are other type ONE_TO_ONE